job_traveler/queries/job_traveler.py

```diff
@@ -15,4 +17,19 @@ def get_items_in_price_list(doctype, txt, searchfield, start, page_len, filters)
             {"txt": "%" + txt + "%", "page_len": page_len, "start": start, "parent": sales_order}
         )
     else:
-        return []
\ No newline at end of file
+        return []
+    
+
+@frappe.whitelist()
+def cutomer_part_number(item, sales_order):
+    current_cutomer =  frappe.get_doc("Sales Order", sales_order)
+    current_cutomer = current_cutomer.customer_name
```

Comment:
`current_customer` please. Also, no need to use a `get_doc`. Use `get_value`.

```frappe.get_value("Sales Order", sales_order, "customer_name")```

It's more efficient